app/views/escorts/confirm_cancel.html.slim

```diff
@@ -0,0 +1,17 @@
+= content_for :breadcrumbs do
```

Comment:
I see your point. The thing is that they are both confirmation pages, but they work differently. In the case of cancelling a PER, you have to provide a reason. In the case of reprint, we just want to make reprint more difficult for users.